### PR TITLE
edly-academy-ui-design-changes

### PR DESCRIPTION
**Pull request by mubeenfayyaz20**
Home Page:

- [x] Container size is bigger than the proposed design (Done)
- [x] Filter dropdown padding is more than the proposed design (Done)
- [x] Small “a” in “All” filter (Done)
- [x] On clicking the second filter when the first one is already in the pressed state, the first one should be closed. Right now both stay open simultaneously.  (Done) 
- [x] Clicking outside should close the filter dropdown. 
- [x] Resource cards are way bigger than the proposed design with extra margins and padding. (Done)
- [x] Icons used in labels are different from the ones provided in the design. 
- [x] Padding in tags is inconsistent. 
- [x]  "What is Edly Academy?” section has too much padding between the heading and paragraph. 
- [x]  Footer is completely different from the proposed design.

Resource Page:

- [x] Videos should also be left aligned like the rest of the content. 
- [x]  There is a lot of padding between the heading and the paragraph. Need to decrease it. 
- [x]  Padding between two sections should be increased. 

Other things:
- [x] Footer design was discussed with you so the rest is good I decreased the spaces on the footer. 
- [x]  Also, I compared all the changes in Figma and encountered many UI issues, but I managed to fix them as well.  (Home and resource page)
- [x]  The mobile responsiveness is good. Also, please set up the hamburger UI popup and elements.

### CODE REVIEW

File: layout/base.html [hunk 1]
```diff
@@ -14,19 +14,24 @@
       href="https://fonts.googleapis.com/css2?family=Inter:wght@100..900&display=swap"
       rel="stylesheet"
     />
+    <style>
```
Comment (regisb) [inline layout/base.html:17]: Why not place these instructions in our academy.css stylesheet?

File: layout/index.html [hunk 2]
```diff
@@ -6,10 +6,11 @@ <h1 class="resources-heading">All Resources</h1>
     <div class="filter">
         <div class="filter-left">
             <p class="filter-text">Filter by:</p>
+            <div class="filter-wrapper">
```
Comment (regisb) [inline layout/index.html:9]: Please fix indentation.

File: layout/static/js/search.js [hunk 3]
```diff
@@ -80,7 +81,27 @@ for (let option of document.getElementsByClassName("category-option")) {
...
+    const resourceTypeButton = document.getElementById('resource-type-button');
+    const resourceTypeFilter = document.getElementById('resource-type-filter');
+  
+    if (!resourceTypeButton.contains(event.target) && !resourceTypeFilter.contains(event.target)) {
+      resourceTypeFilter.classList.add('invisible');
+    }
+  });
+
+
+  document.addEventListener('click', function(event) {
```
Comment (regisb) [inline layout/static/js/search.js:100]: Can we group both event listeners in a single one?

**Review by regisb — CHANGES_REQUESTED** [review]
Awesome! I only have minor comments. Can you please add before/after screenshots in your PR?

File: layout/base.html (hunk 1 above)
Comment (mubeenfayyaz20) [reply layout/base.html:17]: I removed the font-family links for Inter in Edly Academy because we are not using it.

Comment (regisb) [reply layout/base.html:17]: My comment didn't refer to the Inter font-family links, but the `.hs-form ...{...}` rules below. I think they should be moved to the academy.css file.

Comment (mubeenfayyaz20) [reply layout/base.html]: Yes, I also removed it.

**Review by regisb — APPROVED** [review]
awesome, merging now :)

**Pull request merged**